### ISSUE TITLE
NaN scalar logged in tensorboard would crash guild runs info

### ISSUE THREAD

**Issue by wheatdog** [guildai/guildai#211]
```
$ guild check
guild_version:             0.7.0
guild_install_location:    /data/conda/envs/guild-test/lib/python3.6/site-packages/guild
guild_home:                /data/conda/envs/guild-test/.guild
guild_resource_cache:      /data/conda/envs/guild-test/.guild/cache/resources
installed_plugins:         cpu, disk, exec_script, gpu, keras, memory, perf, python_script, queue, skopt
python_version:            3.6.10 |Anaconda, Inc.| (default, May  8 2020, 02:54:21) [GCC 7.3.0]
python_exe:                /data/conda/envs/guild-test/bin/python
platform:                  Linux 5.6.10-arch1-1 x86_64
psutil_version:            5.6.3
tensorboard_version:       2.2.1
cuda_version:              10.2.89
nvidia_smi_version:        440.82
latest_guild_version:      0.7.0

$ cat main.py
# pip install tensorboardX
from tensorboardX import SummaryWriter
import numpy as np

writer = SummaryWriter()

writer.add_scalar('loss', np.nan, 1)

$ guild run main.py -y
Refreshing flags...
WARNING: [root] NaN or Inf found in input tensor.

$ guild runs
[1:feddec10]  main.py  2020-07-23 10:10:29  completed

$ guild runs info 1
Traceback (most recent call last):
  File "/data/conda/envs/guild-test/bin/guild", line 8, in <module>
    sys.exit(main())
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/main_bootstrap.py", line 40, in main
    guild.main.main()
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/main.py", line 33, in main
    _main()
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/main.py", line 40, in _main
    main_cmd.main(standalone_mode=False)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 759, in __call__
    return self.main(*args, **kwargs)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 714, in main
    rv = self.invoke(ctx)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 1132, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 1132, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 951, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/core.py", line 552, in invoke
    return callback(*args, **kwargs)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/external/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/click_util.py", line 151, in fn
    return fn0(*(args + (Args(**kw),)))
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_info.py", line 69, in run_info
    runs_impl.run_info(args, ctx)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 631, in run_info
    _run_info(args, ctx)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 636, in _run_info
    _print_run_info(run, args)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 649, in _print_run_info
    data = _run_info_data(run, args)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 663, in _run_info_data
    data.append(("scalars", _scalar_info(run, args)))
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 696, in _scalar_info
    for key, val in _iter_scalars(run, args)
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 694, in <dictcomp>
    return {
  File "/data/conda/envs/guild-test/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 721, in _iter_scalars
    yield key, "%f (step %i)" % (_s_val(s), _s_step(s))
TypeError: must be real number, not NoneType
```

**Comment by gar1t** [comment]
Thank you! I created a tracking doc for this:

https://github.com/guildai/issue-resolution/tree/master/211-nan-scalar-logged-in-tensorboard-would-crash-guild-runs-info

This will be fixed in 0.7.1.

**Comment by gar1t** [comment]
This is resolved in master is pending the release of 0.7.1.
